src/edgepi/adc/adc_commands.py

```diff
@@ -30,18 +33,31 @@ def write_register_command(self, address, values):
         command = [ADCComs.COM_WREG.value + address, len(values) - 1]
         return command + values
 
-    def start_adc(self, adc_num: ADCNum):
-        """Command to start ADC"""
+    def unsafe_write_register_command(self, address, values):
+        """Trigger ADC register write - unsafe removes all checks"""
+        command = [ADCComs.COM_WREG.value + address, len(values) - 1]
+        return command + values
+
+    def start_adc(self, adc_num: ADCReadInfo):
+        """Command to start ADC conversions"""
         _logger.debug("Command to send is %s", ([adc_num.start_cmd]))
         return [adc_num.start_cmd]
 
-
-    def stop_adc(self, adc_num: ADCNum):
+    def read_adc(self, adc_num: ADCReadInfo):
+        """
+        Returns the command to read from the ADC, after waiting the required time for
+        conversions to take effect
+        """
+        # Since this is full duplex SPI, we have to write something every time we expect
+        # to read something as well, so we add the 6 0xff bytes so that periphery will read
+        # the 6 results bytes we care about.
+        return [adc_num.read_cmd] + [255] * ADC_VOLTAGE_READ_LEN
```

Comment:
I was trying to fit with the pattern `start_adc()` and `stop_adc()` use as well, since the opcode comes from the same location (`adc_num.start_cmd`, `adc_num.read_cmd`, `adc_num.stop_cmd`).

I've added `_command` to all the functions in this module, since it makes for a stronger pattern

https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/424/commits/f41a98a2bce30c6c58d08de32a2ad04041d0dc3b